packages/ag-charts-community/src/chart/series/cartesian/histogramSeriesModule.ts

```diff
@@ -39,13 +34,6 @@ export const HistogramSeriesModule: SeriesModule<'histogram'> = {
             color: DEFAULT_LABEL_COLOUR,
             formatter: undefined,
         },
-        shadow: {
-            enabled: true,
-            color: DEFAULT_SHADOW_COLOUR,
-            xOffset: 0,
-            yOffset: 0,
-            blur: 5,
-        },
```

Comment:
Should we keep this but just set the default to `enabled: false`? What is the OOB shadow enabled behaviour otherwise?

Reply:
No other series has it so why histogram?